wagtail/admin/templates/wagtailadmin/admin_base.html

```diff
@@ -4,6 +4,7 @@
 {% block css %}
     <link rel="stylesheet" href="{% versioned_static 'wagtailadmin/css/vendor/jquery-ui/jquery-ui-1.10.3.verdant.css' %}" />
     <link rel="stylesheet" href="{% versioned_static 'wagtailadmin/css/vendor/jquery.tagit.css' %}">
+    <link rel="stylesheet" href="{% versioned_static 'wagtailadmin/css/vendor/tippy.css' %}">
```

Comment:
I don’t get what this points to – if we do need to load their styles, I’d prefer we do this in Sass via the main `core.css` file or `wagtailadmin.css` / similar "global CSS" files.

Reply:
I might need your help with this one i struggled to path it properly 😣